lib/install/management/appliance.go

```diff
@@ -73,26 +73,15 @@ func (d *Dispatcher) removeApplianceIfForced(conf *metadata.VirtualContainerHost
 	}
 
 	log.Debugf("Appliance is found")
-	if vm != nil && !d.force {
-		err = errors.Errorf("VM already exists with display name %s. Name must be unique. Exiting...", conf.Name)
-		return err
-	}
-
-	// remove it if it's a VCH
 	if ok, verr := d.isVCH(vm); !ok {
 		verr = errors.Errorf("VM %s is found, but is not VCH appliance, please choose different name", conf.Name)
 		return verr
 	}
-	log.Infof("Appliance exists, remove it...")
-
-	var folder string
-	if folder, err = d.deleteVM(vm); err != nil {
-		return err
-	}
-	return d.deleteDatastoreFiles(conf, folder)
+	err = errors.Errorf("Appliance %s exists, to install with same name, please delete it first.", conf.Name)
+	return err
 }
 
-func (d *Dispatcher) deleteVM(vm *vm.VirtualMachine) (string, error) {
+func (d *Dispatcher) deleteVM(vm *vm.VirtualMachine, force bool) (string, error) {
 	var err error
 	power, err := vm.PowerState(d.ctx)
 	if err != nil {
```

Comment:
While it's likely to fail if we cannot get the power state, we should continue our attempt if force == true. Assume that the VM is powered on and continue.